src/styles/stars.scss

```diff
@@ -0,0 +1,29 @@
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 16px;
+  width: 200px;
+}
+
+.card__stars-list {
+  display: flex;
+}
+
+.card__star:not(:last-child) {
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
```

Comment:
The border-radius value of 0.5px is quite small and might not have a noticeable effect. Consider using a larger value if you want the corners to be rounded.